app/questionnaire/router.py

```diff
@@ -64,10 +69,9 @@ def get_next_location_url(self, location, routing_path):
         ):
             return last_block_location.url()
 
-        if self.is_survey_complete():
+        if self.is_survey_complete and not hub_enabled:
             last_section_id = self._schema.get_section_ids()[-1]
             last_block_id = self._schema.get_last_block_id_for_section(last_section_id)
-
             return Location(section_id=last_section_id, block_id=last_block_id).url()
```

Comment:
Existing but this wouldn't technically work for a repeating section but think for now that is fine as we intend to use repeats accompanied by a hub for the time being.